csharp/src/Ice/Communicator.cs

```diff
@@ -133,6 +106,48 @@ private class ObserverUpdater : Instrumentation.IObserverUpdater
         public int DefaultInvocationTimeout { get; }
         public int DefaultLocatorCacheTimeout { get; }
 
+        /// <summary>
+        /// The default context for proxies created using this communicator. Changing the value of
+        /// DefaultContext does not change the context of previously created proxies.
+        /// </summary>
+        public IReadOnlyDictionary<string, string> DefaultContext
+        {
+            get { return _defaultContext; }
+            set { _defaultContext = Value.Count == 0 ? Reference.EmptyContext : new Dictionary<string, string>(value); }
```

Comment:
There is a typo here Value.Count should be value.Count, you should also use expression body for those
```suggestion
            get => _defaultContext;
            set => _defaultContext = value.Count == 0 ? Reference.EmptyContext : new Dictionary<string, string>(value);
```

Reply:
I'll safely assume we always prefer an expression body where possible going forward!
Fixed.

Reply:
Yes we prefer expression body when possible